x-pack/plugins/observability_solution/asset_manager/server/lib/entities/transform/generate_latest_transform.ts

```diff
@@ -28,7 +27,7 @@ export function generateLatestTransform(
       index: `${generateHistoryIndexName(definition)}.*`,
     },
     dest: {
-      index: `${ENTITY_LATEST_BASE_PREFIX}.noop`,
+      index: generateLatestIndexName(definition),
```

Comment:
@simianhacker This looked wrong so I fixed it to write data to a corresponding definition.id data stream. lmk if sending to `noop` was intentional